### PR TITLE
Identity Changelog

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Release History
+
+## 1.0.0-preview.1 (2019-06-28)
+Version 1.0.0-preview.1 is a preview of our efforts in creating an authentication API for Azure SDK client libraries that is developer-friendly, idiomatic
+to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide
+our efforts can be found in the [Azure SDK Design Guidelines for
+.Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
```
Comment (JonathanGiles) [inline sdk/identity/azure-identity/CHANGELOG.md]:
```suggestion
Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
```

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+For release notes and more information please visit https://aka.ms/azure-sdk-preview1-java
```
Comment (JonathanGiles) [inline sdk/identity/azure-identity/CHANGELOG.md]: Fix this link like I suggested for Key Vault

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 1 above)
Comment (g2vinay) [reply sdk/identity/azure-identity/CHANGELOG.md]: Updated

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 2 above)
Comment (g2vinay) [reply sdk/identity/azure-identity/CHANGELOG.md]: Updated

**Pull request merged**